raiden/transfer/mediated_transfer/initiator_manager.py

```diff
@@ -271,17 +348,25 @@ def handle_lock_expired(
 
     Related issue: https://github.com/raiden-network/raiden/issues/3183
 """
-    channel_identifier = payment_state.initiator.channel_identifier
-    channel_state = channelidentifiers_to_channels[channel_identifier]
-    secrethash = payment_state.initiator.transfer.lock.secrethash
+    initiator_state = payment_state.initiator_transfers.get(state_change.secrethash)
+    if not initiator_state:
```

Comment:
what if the route is cancelled? (which is the case for all refunds to the initiator ATM)

Reply:
I think i figured what you're seeing in this PR... clarified it in my previous comment.

Reply:
yep, my bad.